pkg/controllers/nodeclaim/consistency/controller.go

```diff
@@ -77,6 +78,8 @@ func NewController(clk clock.Clock, kubeClient client.Client, cloudProvider clou
 
 func (c *Controller) Reconcile(ctx context.Context, nodeClaim *v1.NodeClaim) (reconcile.Result, error) {
 	ctx = injection.WithControllerName(ctx, "nodeclaim.consistency")
+	ctx = log.IntoContext(ctx, log.FromContext(ctx).WithValues("Node", klog.KRef("", nodeClaim.Status.NodeName)))
+
```

Comment:
Move this after the `IsManaged` block?

Reply:
I could go either way -- yes, it could save a tiny amount of time, but I do prefer that we keep all of the context injection sort-of together at the top of the Reconcile()